onnxruntime/core/framework/allocator.cc

```diff
@@ -14,10 +13,12 @@ void* CPUAllocator::Alloc(size_t size) {
     return nullptr;
   //default align to 64;
   void* p;
-#ifdef _WIN32
+#if defined(__AVX512F__)
+  size_t alignment = 64;
+#elif defined(__AVX__)
```

Comment:
do we need 2 else blocks if they do the same exact thing?

Reply:
Not really. But I think in this way it's more readable.  It's easier to add a comment to say why we are doing this.